client/gutenberg/extensions/subscriptions/edit.jsx

```diff
@@ -0,0 +1,166 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { Component, Fragment } from '@wordpress/element';
+import {
+	TextControl,
+	Button,
+	ToggleControl,
+	Disabled,
+	withFallbackStyles,
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+import { sprintf } from '@wordpress/i18n/build/index';
+import apiFetch from '@wordpress/api-fetch';
+import { compose } from '@wordpress/compose';
+import {
+	ContrastChecker,
+	InspectorControls,
+	withColors,
+	PanelColorSettings,
+} from '@wordpress/editor';
+
+const { getComputedStyle } = window;
```

Comment:
Since we already use `withFallbackStyles`, are we planning to use this anywhere?